src/signed.rs

```diff
@@ -54,7 +54,7 @@ pub trait Signable {
 /// `[PartiallyMultisigned<'a, T, MK>]`.
 #[derive(Clone, Debug, Decode, Encode)]
 pub struct UncheckedSigned<T: Signable, S> {
-    signable: T,
+    pub(crate) signable: T,
```

Comment:
Getter, please?

Reply:
Kind of related: `Multisigned` has `unchecked` public, but it would be nicer if it did the same as `Signed`, i.e. used `into`.

Reply:
Done